tests/drivers/clock_control/stm32_clock_configuration/stm32n6_devices/src/test_stm32_clock_configuration_i2c.c

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2022 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/ztest.h>
+#include <soc.h>
+#include <zephyr/drivers/clock_control.h>
+#include <zephyr/drivers/clock_control/stm32_clock_control.h>
+#include <zephyr/logging/log.h>
```

Comment:
I wonder if we actually need log.h.

Reply:
Sorry, It's a leftover from some tests I previously made. I'll remove, here and at few other places.